janus_server/src/config.rs

```diff
@@ -38,6 +39,8 @@ pub struct AggregatorConfig {
     pub listen_address: SocketAddr,
     /// The aggregator's database configuration.
     pub database: DbConfig,
+    /// The aggregator's public HTTPS endpoint, which will be advertised in error responses.
+    pub https_endpoint: Url,
```

Comment:
Shouldn't this come from the task parameters, since HPKE config and the rest of the API has to be advertised relative to the aggregator endpoint there?

Reply:
Based on the current text of the interop draft, I don't think so, as one HTTPS endpoint may service multiple tasks. It's true that this endpoint should match either the first or second endpoint in each task's configuration, but I think it still makes sense to provide this as part of the server-wide configuration, before the first task is loaded in.

Reply:
I don't agree -- this just introduces the possibility that the URL in the config will disagree with the URL in the task parameters. In any case, [RFC 7807 section 3.1](https://datatracker.ietf.org/doc/html/rfc7807#section-3.1) makes it clear we can provide a relative URI here, which means we can avoid putting absolute URIs here at all. But we can revisit this as protocol text evolves.

Reply:
Good point! I like that